### PR TITLE
Multi-tenancy fixes

### PR DESCRIPTION
**Pull request by OmarAyo**
this PR fixes some wrong statements/commands in our current multi-tenancy doc

### CODE REVIEW

File: content/enterprise-features/multitenancy.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ For multi-tenant environments a suggested `query-limit` value is 500ms.
 
 {{% notice "note" %}}
 Only super-admins ([Guardians of the galaxy](#guardians-of-the-galaxy)) have access across tenants.
-The super admin is used only for database admininstration operations, such as exporting data of all tenants. 
+The super admin is used only for database admininstration operations, such as exporting data of all tenants. Guardian of the galaxy cannot read across tenats.
```
Comment (NamanJain8) [inline content/enterprise-features/multitenancy.md:44]: typo `tenants`

File: content/enterprise-features/multitenancy.md [hunk 2]
```diff
@@ -81,8 +81,6 @@ Multi-tenancy defines certain ACL roles for the shared cluster:
 
 - [Guardians of the Galaxy](#guardians-of-the-galaxy) (Super Admins) 
 - Guardians of the Namespace
-  - They can add users to groups inside the namespace
```
Comment (NamanJain8) [inline content/enterprise-features/multitenancy.md:84]: Why are we removing this? Guardian of a namespace can create users within their namespace.

**Review by NamanJain8 — CHANGES_REQUESTED** [review]

File: content/enterprise-features/multitenancy.md (hunk 2 above)
Comment (OmarAyo) [reply content/enterprise-features/multitenancy.md:84]: @NamanJain8 AFAIK they can only create a namespace that will also create the guardian of namespace group that has the groot user. IDK if the guardian of galaxy users can create other groups or users in other namespaces

File: content/enterprise-features/multitenancy.md (hunk 1 above)
Comment (OmarAyo) [reply content/enterprise-features/multitenancy.md]: fixed

File: content/enterprise-features/multitenancy.md (hunk 2 above)
Comment (ahsanbarkati) [reply content/enterprise-features/multitenancy.md:84]: That's right. They cannot create users for other namespaces but within the galaxy namespace, they can do anything that a guardian group can do. It includes creating users.

Comment (OmarAyo) [reply content/enterprise-features/multitenancy.md:84]: ok then let me rephrase it better to clarify that they can only create users/groups within their namespace

File: content/enterprise-features/multitenancy.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ For multi-tenant environments a suggested `query-limit` value is 500ms.
 
 {{% notice "note" %}}
 Only super-admins ([Guardians of the galaxy](#guardians-of-the-galaxy)) have access across tenants.
-The super admin is used only for database admininstration operations, such as exporting data of all tenants. 
+The super admin is used only for database admininstration operations, such as exporting data of all tenants. Guardian of the galaxy users cannot read across tenants.
```
Comment (bucanero) [inline content/enterprise-features/multitenancy.md:44]:
```suggestion
The super admin is used only for database administration operations, such as exporting data of all tenants. _Guardian of the Galaxy_ members cannot read across tenants.
```

File: content/enterprise-features/multitenancy.md [hunk 4]
```diff
@@ -81,9 +81,13 @@ Multi-tenancy defines certain ACL roles for the shared cluster:
 
 - [Guardians of the Galaxy](#guardians-of-the-galaxy) (Super Admins) 
 - Guardians of the Namespace
+  - They can create/add users/groups inside their own namespace
```
Comment (bucanero) [inline content/enterprise-features/multitenancy.md:84]:
```suggestion
  - They can create users and groups inside their own namespace
```

File: content/enterprise-features/multitenancy.md [hunk 5]
```diff
@@ -81,9 +81,13 @@ Multi-tenancy defines certain ACL roles for the shared cluster:
...
+  - They can assign users to groups inside their own namespace
+  - They can assign predicates to groups inside their own namespace
   - They can add users to groups inside the namespace
-  - They can remove users from groups inside the namespace
-  - They can export their namespace
+  - They can export their namespace 
+  - They can query/mutate in their namespace
```
Comment (bucanero) [inline content/enterprise-features/multitenancy.md:89]:
```suggestion
  - They can query and mutate in their namespace
```

File: content/enterprise-features/multitenancy.md [hunk 6]
```diff
@@ -495,11 +498,22 @@ A namespace-specific export will contain the namespace value in the generated `.
...
 mutation {
-  export(input: {format: "rdf", namespace: 0x1234}) {
+  export(input: {format: "rdf", namespace: 1234}) {
+    response {
+      message
+    }
+  }
+}
+```
+When using the _Guardian of the Namespace_, then no need to specify the namespace in the GraphQL mutation, as they can only export within their own namespace:
```
Comment (bucanero) [inline content/enterprise-features/multitenancy.md:512]:
```suggestion
When using the _Guardian of the Namespace_, there's no need to specify the namespace in the GraphQL mutation, as they can only export within their own namespace:
```

File: content/deploy/fast-data-loading/live-loader.md [hunk 7]
```diff
@@ -93,7 +93,7 @@ Using `--force-namespace`, a _Guardian_ can load the data into the namespace spe
 The Live loader requires that the `namespace` from the data and schema files exist before loading the data.
 {{% /notice %}}
 
-For example, to preserve the namespace while loading data:
+For example, to preserve the namespace while loading data first you need to create the namespace/s and then run the live loader command:
```
Comment (bucanero) [inline content/deploy/fast-data-loading/live-loader.md:96]:
```suggestion
For example, to preserve the namespace while loading data first you need to create the namespace(s) and then run the live loader command:
```

**Review by NamanJain8 — APPROVED** [review]
Looks good. Thanks @OmarAyo !! 🎉

**Review by ahsanbarkati — APPROVED** [review]
Awesome. Thanks Omar

**Review by bucanero — APPROVED** [review]
